### ISSUE TITLE
Add random appearing rewards

### ISSUE THREAD

**Issue by BrianFreemanAtlanta** [ignacioxd/ragin-mages#133]
Add rewards that randomly appear and characters can pick up
- healing potions 
- power burst (strong/long shot for x time)
- shield - reduce damage or invulnerable for x time
- boots of speed - faster for x time
- others?

**Comment by PepperAddict** [comment]
I like the healing potions idea, but I'm a bit on the no-no on the boosts idea. I've played games that offer that kind of thing and it got boring or annoying fast. Before you know it, ragin mages might end up being a pay-to-win game. "spend 5 dollars on this boost package" *cough* agar *cough*.
 
That's just my negative opinion though. What does everybody else think about this?

**Comment by ignacioxd** [comment]
I agree that we should be careful about pay-to-win scenarios. I'm not sure this is a problem if the game is 100% free. I do see a problem with picking up stuff in that faster characters have an advantage over slower ones.

Maybe we can reward some players based on their performance. For example, when you kill someone else, you may absorb their life essence and heal a bit. The more you kill the more you heal. Players who are more accurate can get a power burst (e.g., if you kill 3 people in a row with 100% accuracy you gain strength+range until you lose that accuracy).
